child/_child_terra_sign_in.Rmd

```diff
@@ -0,0 +1,7 @@
+You need to sign into Terra to establish your account. [Launch Terra](https://anvil.terra.bio/#workspaces), and you should be prompted to sign in with your Google account.
+
+You can always access Terra by going to [`anvil.terra.bio`](https://anvil.terra.bio/), or by clicking the link on the AnVIL home page.
+
+```{r, echo=FALSE, fig.alt='Screenshot of the AnVIL home page. The button to "launch Terra" is highlighted.'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1-PVlDAzmDncjcCHIy9ZyR3ROyPpfjrg46LhEQopDc1Q/edit#slide=id.g22c9add987a_0_5")
```

Comment:
Thanks for updating the screenshot!

Consider making a new slidedeck here https://drive.google.com/drive/folders/1LmaBTt4vgs9lMW-16fYjWVk64vegzd52 - I tried to keep the slide decks for the child documents together in one place for easier maintenance.